setup.py

```diff
@@ -129,7 +133,7 @@ def get_versions(default={}, verbose=False):
     packages=find_packages('src', exclude=['leap.keymanager.tests']),
     package_dir={'': 'src'},
     test_suite='leap.keymanager.tests',
-    install_requires=utils.parse_requirements(),
```

Comment:
hmm no, it doesn't work this way. the function get_reqs from files shortcircuits as soon as one of the alternatives is found in the filesystem. 
I'll update the docstring